docker/templates/smtp-notifier.tmpl

```diff
@@ -1,4 +1,4 @@
-To: {{.To}}
+To: {{range $index, $v := .To}}{{if $index}},{{end}}{{$v}}{{end}}
```

Comment:
Nice way to handle `,`. 🙂

Reply:
@dusanb94 I got the idea fro [here](https://groups.google.com/g/golang-nuts/c/XBScetK-guk/m/Bh7ZFz6R3wQJ) :)